### DISCUSSION TITLE
how popular is "very popular"?

### DISCUSSION THREAD

**Discussion by dankamongmen** [Homebrew/discussions#1893] [Tap maintenance and Homebrew development] [answered]
Hey there! I'm the author of a library that has until recently been available only on Linux and the BSDs (it's been in active development since November 2019). I recently added support for macOS, and have prepared a formula which i believe to be assembled according to recommended principles. I was going to submit it, but noticed the line "We frown on authors submitting their own work unless it is very popular" on  https://docs.brew.sh/Acceptable-Formulae. I was hoping for some guidance. Obviously it's not too popular on macOS, as it only very recently runs there. I'm not sure how to evaluate popularity overall, but it's present in at least a dozen Linux distributions, and has ~1K github stars, for whatever that's worth.

i'm hoping to pick up macOS usage, which would definitely be facilitated by inclusion in the main Homebrew repository. it's very actively maintained, and has a fairly healthy community.

ought/may i submit the brew for inclusion? thanks!

**Comment by SMillerDev** [comment, marked as answer]
I'd say the fact that it was picked up by Linux distributions already qualifies it. That line is mostly to discourage people who spent a weekend writing a hello world app and now want to ship it in brew.

**Reply by dankamongmen** [reply]
got it, that's exactly what i was hoping. https://github.com/dankamongmen/notcurses great i'll go ahead and PR the recipe, thanks a ton!
